docs/headless-cms/08-01-add-new-documentation-to-console.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Add new documentation to the Console's documentation view
+type: Tutorial
+---
+
+This tutorial shows how you can customize the Documentation view that is available in the Console UI under the ![](./assets/docs.png) icon on the top navigation panel. The purpose of this tutorial is to create a new **Prometheus** documentation section that contains **Concepts** and **Guides** documentation topics with a set of Markdown subdocuments. The Markdown sources used in this tutorial point to specific topics in the official Prometheus documentation.  
+
+When you prepare ClusterDocsTopics, make sure that:
+- ClusterDocsTopics have a proper [custom resource definition](#custom-resource-clusterdocstopic).
+- Documents start with metadata, including obligatory `Title` for the document name and optional `Type` for any additional document grouping.
+- Documents are written in Markdown and follow a simple Markdown syntax. Any customizations can result in incorrect images or links rendering. To avoid such issues, use our [content guidelines](https://github.com/kyma-project/community/tree/master/guidelines/content-guidelines).
+
+
+## Prerequisites
+
+- Kyma
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+## Steps
+
+1. Open the terminal and apply these ClusterDocsTopic custom resources:
+
+```yaml
+cat <<EOF | kubectl apply -f -
+apiVersion: cms.kyma-project.io/v1alpha1
+kind: ClusterDocsTopic
+metadata:
+  labels:
+    cms.kyma-project.io/view-context: docs-ui # This label specifies that you want to render documents in the Documentation view.
+    cms.kyma-project.io/group-name: prometheus # This label defines the group under which you want to render the given asset in the Documentation view. The value cannot include spaces.
+    cms.kyma-project.io/order: "1" # This label specifies the position of the ClusterDocsTopic in relation to other ClusterDocsTopics in the Prometheus section.
+  name: prometheus-concepts
+spec:
+  displayName: "Concepts" # The name of the topic that shows in the Documentation view under the main Prometheus section.
+  description: "Some docs about Prometheus concepts"
+  sources:
+    - type: markdown # This type indicates that the Asset Metadata Service must extract front matter metadata from the source Prometheus documents and add them to a ClusterDocsTopic as a status.
+      name: docs
+      mode: package # This mode indicates that you import a ZIP file instead of a single file.
```

Comment:
This indicates that controller knows the source is a pckage that first must be unpacked for further processing